### DISCUSSION TITLE
POS into the json file

### DISCUSSION THREAD

**Discussion by JessitaMS** [tecoholic/ner-annotator#40] [General]
Hi Arun:

I have a question for you, if we use this amazing tool to mark entities, how can we put the POS of the text, because we only put the entities in the json file? Do you have another tool that does what I told you about?

Thanks!

**Comment by tecoholic** [comment]
@JessitaMS I haven't worked with POS Tagging. So didn't build this tool with POS tagging in mind. Sorry, I don't know of any other tool either. If you find something kindly share with me, I will try to add the functionality into the NER Annotator.
